src/chrome/content/rules/Github-Pages.xml

```diff
@@ -15,9 +15,8 @@
 		<test url="http://duckietv.github.io/DuckieTV" />
 		<test url="http://duckietv.github.io/DuckieTV/" />
 
-	<!-- See https://github.com/EFForg/https-everywhere/issues/2507 -->
-	<exclusion pattern="^http://googletrends\.github\.io/(GOPDebate1|iframe-scaffolder)/" />
-		<test url="http://googletrends.github.io/GOPDebate1/" />
+	<exclusion pattern="^http://googletrends\.github\.io/iframe-scaffolder/?" />
```

Comment:
Why not use the simpler

```suggestion
	<exclusion pattern="^http://googletrends\.github\.io/iframe-scaffolder" />
```

?

Reply:
@Bisaloo my mistake, changed now thanks!